### PR TITLE
Adding pgsu-feedstock

### PR DESCRIPTION
**Pull request by ltalirz**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pgsu/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [ py < 27 or py > 38 ]
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - click
+    - enum34;   # [ py < 35 ]
```
Comment (marcelotrevisani) [inline recipes/pgsu/meta.yaml:25]:
```suggestion
    - enum34   # [ py < 35 ]
```

Comment (marcelotrevisani) [reply recipes/pgsu/meta.yaml:25]: Actually, you can remove this deps, we just create packages for Python 2.7 and 3.6+

File: recipes/pgsu/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  number: 0
+  entry_points:
+    - pgsu=pgsu.cli:run
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [ py < 27 or py > 38 ]
+
+requirements:
+  host:
+    - pip
+    - python
```
Comment (marcelotrevisani) [inline recipes/pgsu/meta.yaml:22]:
```suggestion
    - python >3.0,<3.8
```

File: recipes/pgsu/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: af33bc917b26cede741f53953f54e5cef9d6428bd56a68f37198ac7879e99e83
+
+build:
+  number: 0
+  entry_points:
+    - pgsu=pgsu.cli:run
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [ py < 27 or py > 38 ]
```
Comment (marcelotrevisani) [inline recipes/pgsu/meta.yaml:17]:
```suggestion
  noarch: python
```

File: recipes/pgsu/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - click
+    - enum34;   # [ py < 35 ]
+    - psycopg2 >=2.8.3
+    - python
```
Comment (marcelotrevisani) [inline recipes/pgsu/meta.yaml:27]:
```suggestion
    - python  >3.0,<3.8
```

File: recipes/pgsu/meta.yaml (hunk 1 above)
Comment (marcelotrevisani) [inline recipes/pgsu/meta.yaml:25]:
```suggestion
    - enum34   # [py<35]
```

**Comment by marcelotrevisani** [comment]
If you want to, next time you might give a go to [Grayskull :skull: ](https://github.com/marcelotrevisani/grayskull/) to generate the python recipes :smile:

**Comment by ltalirz** [comment]
thanks, I'll check it out next time

**Comment by ltalirz** [comment]
@marcelotrevisani Would you mind merging this PR?
Is there anything else I need to do here?

**Comment by ltalirz** [comment]
Thanks!

**Comment by marcelotrevisani** [comment]
> Thanks!

Sorry for the delay!

**Pull request merged**